### ISSUE TITLE
Change form to wizzard

### ISSUE THREAD

**Issue by ligi** [ethereum/sourcify#102]
This is just an idea for discussion. Maybe we should change the form to a wizzard. 
This would mean we would ask for the chain as step 1 then ask for the address as step 2 and then ask for the metadata in the final step 3
Advantages I see:
 - IMHO improved UX
 - after step 2 we could check if we already have this contract verified and skip step 3. This could also be a nice way to have a UI to access the metadata
 - after step2 we could try to fetch the metadata via ipfs/swarm and maybe make step3 obsolete in some cases

**Comment by FabijanC** [comment]
Closing because this was superseded by other issues such as https://github.com/ethereum/sourcify/issues/345 and https://github.com/ethereum/sourcify/issues/336
